base/operators.jl

```diff
@@ -68,12 +68,16 @@ Supertype operator, equivalent to `T2 <: T1`.
 """
     supertype(T::DataType)
 
-Return the supertype of DataType `T`.
+Return the supertype of type `T`.
```

Comment:
Don't use the definite article here, a type may have more than one supertype. Not sure what the best wording is.

Reply:
I've seen direct supertype around (e.g. [oracle docs](https://docs.oracle.com/javase/specs/jls/se21/html/jls-4.html)), would that work?